content/docker-hub/api/latest.yaml

```diff
@@ -102,8 +102,8 @@ tags:
   - name: repositories
     x-displayName: Repositories
     description: |
-      The repository endpoints allow you to manage your repository's
-      configuration like description.
+      The repository endpoints allow you to access your repository's
```

Comment:
**Note to reviewer:** I updated this because we only list endpoints related to tags.